### DISCUSSION TITLE
disable auto scrolling when moving objects

### DISCUSSION THREAD

**Discussion by MattesMrzik** [xournalpp/xournalpp#3247] [Q&A] [answered]
Hi,
as part of note-taking I insert some screenshots into my .xopp file. When I want to fine tune their position on the page, I sometimes touch the border while dragging the object. Then xournalpp scrolls up and down when I touch the upper and lower bound respectively. But I prefer it to not do this, like it did in older version of the application. Is there a way you know of to achieve this behavior?
Thanks

**Comment by Technius** [comment, marked as answer]
This is a bug, see #2889. I'm hoping to fix this by the next minor release (~2 weeks-1 month).

**Reply by Technius** [reply]
This bug will be fixed in the next version `1.1.1`.
